plugins/console/spec/consoleFailLogWarnings.js

```diff
@@ -0,0 +1,14 @@
+var env = require('../../../spec/environment.js');
```

Comment:
Ah I didn't realize that that's where they were run from. I'll get them added to that and update the PR in a little bit.

Thanks for the feedback!